src/Attributes.elm

```diff
@@ -178,7 +178,7 @@ getDescription attribute value =
     in
         (List.filter (isLessThanAttribute value) attributeDescriptions)
             |> List.head
-            |> Maybe.map snd
+            |> Maybe.map Tuple.second
```

Comment:
Makes it much easier for someone who doesn't know snd to understand. Be as verbose as necessary. I think this is actually *perfectly concise*